op-service/eth/types.go

```diff
@@ -292,7 +301,7 @@ func (envelope *ExecutionPayloadEnvelope) CheckBlockHash() (actual common.Hash,
 	return blockHash, blockHash == payload.BlockHash
 }
 
-func BlockAsPayload(bl *types.Block, shanghaiTime *uint64) (*ExecutionPayload, error) {
+func BlockAsPayload(bl *types.Block, config *params.ChainConfig) (*ExecutionPayload, error) {
```

Comment:
We now need to check for `IsthmusTime` as well before setting `WithdrawalsRoot` in the payload, so we pass in `ChainConfig`.